database/test_database.py

```diff
@@ -103,6 +121,63 @@ def test_delete_and_save_directory(setup_mock_directory):
         saved_directory = json.load(f)
         assert saved_directory == mock_directory
 
+from unittest.mock import patch
+
+@patch('console.show_options_menu', return_value='mascotas')
+def test_access_existing_database(mock_show_options_menu, setup_mock_directory):
+    "Access: Debe acceder correctamente a una base de datos existente"
+    mock_directory, _ = setup_mock_directory
```

Comment:
```suggestion
    mock_directory, _ = setup_mock_directory
```
setup_mock_directory pertenece a pytest,
mientras que @patch(...) pertenece a unittests

mezclar frameworks no suele ser una buena idea